server/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -328,6 +328,9 @@ private ProgramCardFactory.ProgramCardData buildProgramCardData(
         draftRowExtraActions.add(maybeManageTranslationsLink.get());
       }
       draftRowExtraActions.add(renderEditStatusesLink(draftProgram.get()));
+      if (featureFlags.getFlagEnabled(request, PUBLISH_SINGLE_PROGRAM_ENABLED)) {
+        draftRowExtraActions.add(renderPublishProgramLink(draftProgram.get(), request));
```

Comment:
**suggestion (non-blocking):** (you mention that the UI is tentative and this is feature flagged so this is non-blocking) could disable the button if the program has modified questions shared with other programs